torch/masked/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .maskedtensor.core import is_masked_tensor, MaskedTensor
```

Comment:
In which case should you use `MaskedTensor` or `masked_tensor`?

Reply:
I don't have a great answer for this - I've been using `masked_tensor` primarily in tutorials (i.e. externally). Is there a specific usage pattern that you'd recommend?

Reply:
Yeah that makes sense, thanks!